docs/cli-usage.md

```diff
@@ -150,6 +150,46 @@ Identifier styles (ex: `FooBar`, `fooBar`, `foo_bar`, `FOO_BAR`, `m_fooBar`)
 | `--ident-objc-type-param ...` |
 | `--ident-objc-local ...` |
 | `--ident-objc-file ...` |
+|   |
 
+The _Identifier Style_ is a text conversion hint on how to generate names from the djinni idl file.
 
+Example:
+The djinni idl for an enum
 
+```
+SomeEnum = enum {
+    Value;
+}
+```
+
+transforms per default to the following Java code
+
+```
+public enum SomeEnum {
+    VALUE,
+    ;
+}
+```
+As you see, _VALUE_ is now in upper case letters.
+
+If you use `--ident-java-enum foo_bar` then _Value_ will not be transformed into upper case and the following Java code will be generated.
```

Comment:
This is so weird 🙈  
If changing the Identifier names would not potentially break so many builds I'd prefer more speaking names for them:

- `camelCase`
- `TrainCase`
- `snake_case`
- `none` <== This is what `foo_bar` currently is.
- ...

Reply:
What do you think about renaming them? This just came to my mind a few days ago. I don't see it in the scope of this PR.

Reply:
might still be a breaking change. 
we could add `none` as a keyword to start a smooth transition process ....

Reply:
just want to talk with you about this on a theoretical basis. :P I'm aware that this is a major breaking change.